src/arrow2/src/io/parquet/mod.rs

```diff
@@ -22,6 +22,9 @@ impl From<parquet2::error::Error> for Error {
             parquet2::error::Error::Transport(msg) => {
                 Error::Io(std::io::Error::new(std::io::ErrorKind::Other, msg))
             }
+            parquet2::error::Error::IoError(msg) => {
+                Error::Io(std::io::Error::new(std::io::ErrorKind::Other, msg))
+            }
```

Comment:
makes sense to me... Will let @samster25 do a sign-off

Reply:
this looks good.

Reply:
Minor nit; could instead be:
```rs
[par](parquet2::error::Error::Transport(msg)) | parquet2::error::Error::IoError(msg) => ...
```

But I mean doesn't matter too much. I'm good with current.

Reply:
unfortunately one of the `msg` is `String`, and the other is an `std::io::Error` :')